olp-cpp-sdk-dataservice-read/src/VersionedLayerClientImpl.h

```diff
@@ -81,6 +81,14 @@ class VersionedLayerClientImpl {
   virtual client::CancellableFuture<PrefetchTilesResponse> PrefetchTiles(
       PrefetchTilesRequest request, PrefetchStatusCallback status_callback);
 
+  virtual client::CancellationToken PrefetchPartitions(
+      PartitionsRequest request, PrefetchPartitionsResponseCallback callback,
+      PrefetchStatusCallback status_callback);
```

Comment:
PrefetchStatus gives you the following information: `prefetched_tiles` and `total_tiles_to_prefetch`.
I don't think this is suitable for a partitions prefetch. Please create another struct, e.g. `PrefetchPartitionsStatus` in `PrefetchStatus.h´ and use that.